examples/blocks.js

```diff
@@ -245,13 +245,20 @@ registerBlockType( 'gb-m-example/simple-block', {
 			},
 		},
 		columns: {
-			type: 'string',
+			type: 'array',
 			field: {
```

Comment:
I think it should not be `array`, @sayedtaqui  can you please confirm.

Reply:
@yahilmadakiya It will be array of objects.

Reply:
@vaishaliagola27 Why would this be array of objects? 🤔